lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -151,6 +152,9 @@ def extract_student(student, features):
             for meta_feature, meta_key in meta_features:
                 student_dict[meta_feature] = meta_dict.get(meta_key)
 
+            if include_city_column and 'city' in meta_dict:
```

Comment:
@kaizoku, this doesn't fall back to `UserProfile.city` as @kdmccormick had recommended. If there isn't a reason for doing so, can you update this?

> If not, I'd support a PR that made the report first check UserProfile.meta, then falling back to UserProfile.city if it is missing/falsy. For good measure you could even leave a comment on UserProfile.city saying that it is deprecated in favor of of a field on UserProfile.meta.

Reply:
@lgp171188 , thanks for the catch. I tested with an empty UserProfile.meta but forgot to test with UserProfile.meta["city"] set to an empty string or the like.

This falls back to UserProfile.city if UserProfile.meta has no key for "city", but doesn't check if the key is falsy. I updated the PR to fall back to UserProfile.city if UserProfile.meta['city'] is an empty string or is completely unset.